File: app/src/main/java/com/kra/paypoint/data/remote/model/master/ItemSaveReq.kt
package com.kra.paypoint.data.remote.model.master

import com.google.gson.annotations.SerializedName

data class ItemSaveReq(
    @SerializedName("tin") val tin: String,
    @SerializedName("bhfId") val bhfId: String,
    @SerializedName("itemCd") val itemCd: String,
    @SerializedName("itemClsCd") val itemClsCd: String,
    @SerializedName("itemTyCd") val itemTyCd: String,
    @SerializedName("itemNm") val itemNm: String,
    @SerializedName("itemStdNm") val itemStdNm: String?,
    @SerializedName("orgnNatCd") val orgnNatCd: String?,
    @SerializedName("pkgUnitCd") val pkgUnitCd: String,
    @SerializedName("qtyUnitCd") val qtyUnitCd: String,
    @SerializedName("taxTyCd") val taxTyCd: String,
    @SerializedName("btchNo") val btchNo: String?,
    @SerializedName("bcd") val bcd: String?,
    @SerializedName("dftPrc") val dftPrc: Double,
    @SerializedName("grpPrcL1") val grpPrcL1: Double?,
    @SerializedName("grpPrcL2") val grpPrcL2: Double?,
    @SerializedName("grpPrcL3") val grpPrcL3: Double?,
    @SerializedName("grpPrcL4") val grpPrcL4: Double?,
    @SerializedName("grpPrcL5") val grpPrcL5: Double?,
    @SerializedName("addInfo") val addInfo: String?,
    @SerializedName("sftyQty") val sftyQty: Double?,
    @SerializedName("useYn") val useYn: String,
    @SerializedName("regBhfId") val regBhfId: String?,
    @SerializedName("rraModYn") val rraModYn: String?,
    @SerializedName("isrcAplcbYn") val isrcAplcbYn: String?,
    @SerializedName("batchNum") val batchNum: String?,
    @SerializedName("regrId") val regrId: String?,
    @SerializedName("regrNm") val regrNm: String?,
    @SerializedName("modrId") val modrId: String?,
    @SerializedName("modrNm") val modrNm: String?
)

File: scratch/EBM2x_legacy/RraSdc/model/ItemSaveReq.cs
using System;
using System.Collections.Generic;

namespace EBM2x.RraSdc.model
{
    public class ItemSaveReq {
	    public string tin { get; set; }  // TIN
	    public string bhfId { get; set; }  // Branch ID
	    public string itemCd { get; set; }  // Item Code
	    public string itemClsCd { get; set; }  // Item Classification Code
	    public string itemTyCd { get; set; }  // item Type Code
	    public string itemNm { get; set; }  // item Name
	    public string itemStdNm { get; set; }  // Item Standard Name
	    public string orgnNatCd { get; set; }  // Origin Place Code (Nation)
	    public string pkgUnitCd { get; set; }  // Packaging Unit Code
	    public string qtyUnitCd { get; set; }  // Quantity Unit Code
	    public string taxTyCd { get; set; }  // Taxation Type Code
	    public string btchNo { get; set; }  // Batch Number
	    public string bcd { get; set; }  // Barcode
	    public double dftPrc { get; set; }  // Default Unit Price
	    public double grpPrcL1 { get; set; }  // Group1 Unit Price
	    public double grpPrcL2 { get; set; }  // Group2 Unit Price
	    public double grpPrcL3 { get; set; }  // Group3 Unit Price
	    public double grpPrcL4 { get; set; }  // Group4 Unit Price
	    public double grpPrcL5 { get; set; }  // Group5 Unit Price
	    public string addInfo { get; set; }  // Add Information
	    public double sftyQty { get; set; }  // Safty Quantity
	    public string useYn { get; set; }  // Used / UnUsed
	    public string regBhfId { get; set; }  // Registrant Branch ID
        public string rraModYn { get; set; }  //RraModYn
        public string isrcAplcbYn { get; set; }  //IsrcAplcbYn
        public string batchNum { get; set; }  // BatchNum
        public string regrId { get; set; }  //   등록자 아이디
        public string regrNm { get; set; }  //    등록장 명
        public string modrId { get; set; }  //   수정자 아이디
        public string modrNm { get; set; }  //    수정자 명

    }
}
